Samples/2a-validation/shell/stderr.txt

```diff
@@ -1,3 +1,14 @@
+
```

Comment:
I tweaked the colorization of the messages to make them pop a bit.

Reply:
(Note the clever highlighting of the filename/row/column and the red/yellow for error/warning)